src/components/conversation/cursor.js

```diff
@@ -74,24 +74,56 @@ const getCaretData = (el, position) => {
 /*
   Sets the caret position.
   Parameters:
-    d, Object{node, offset}: where node is the Node that contains the caret and
-      offset is the position within the Node
     el, DOM Element: the element that contains the Node
     position, int: the desired position
 */
-const setCaretPosition = (d, el) => {
+const setCaretPosition = (el, position) => {
   const doc = el.ownerDocument || el.document;
   const win = doc.defaultView || doc.parentWindow;
   const sel = win.getSelection();
   const range = doc.createRange();
-  range.setStart(d.node, d.offset);
+
+  const cd = getCaretData(el, position);
+  if (cd.node === undefined) {
+    cd.node = el;
+  }
+  range.setStart(cd.node, cd.offset);
   range.collapse(true);
   sel.removeAllRanges();
   sel.addRange(range);
 };
 
+/*
+  Sets the active user's displayed caret position.
+  Parameters:
+    el, DOM Element: the element that contains the Node
+    position, int: the desired position
+    id, int: the active user's id
+    colour, string: the colour to use for the caret
+*/
+const setActiveUserCaretPosition = (el, position, id, colour) => {
+  console.log(position, id, colour);
```

Comment:
I think this should be `debugLog()` instead?

Reply:
Whoops! Yeah I forgot haha.